Import collision name clash

Leaving it up here for people who face the same problem as me.
Also, I couldn't submit a pull request, so hoping the author sees this and makes a small change to his code.

If you're going through the Sensorbee setup [tutorial](http://docs.sensorbee.io/en/latest/tutorial.html#id2) and are getting the following error:
`../../go/src/github.com/sensorbee/fluentd/plugin/plugin.go:4:2: case-insensitive import collision: "github.com/Sirupsen/logrus" and "github.com/sirupsen/logrus"`
Go to:
`$GOPATH/go/src/github.com/sensorbee/fluentd/source.go` and change the line of code that says
`"github.com/Sirupsen/logrus"` to `"github.com/sirupsen/logrus"`.

Essentially, what's happening is, if you check your `$GOPATH/src/github.com` you'll see that you're generating two files:
`sirupsen` and `Sirupsen`, when you run build_sensorbee, and this causes a conflict with the case-insensitive build system.

Now, the thing is this [issue](https://github.com/osrg/gobgp/pull/1182), `Sirupsen` has been renamed to `sirupsen`

EDIT: You might also want to delete `Sirupsen` before you try building again.

--- a/source.go
+++ b/source.go
@@ -1,7 +1,7 @@
 package fluentd
 
 import (
-	"github.com/Sirupsen/logrus"
+	"github.com/sirupsen/logrus"
 	"github.com/fluent/fluentd-forwarder"
 	"github.com/op/go-logging"
 	"gopkg.in/sensorbee/sensorbee.v0/bql"

